src/start_hermes.js

```diff
@@ -17,10 +17,16 @@ async function start(logger) {
   const builder = new Builder();
   await builder.build(config);
   await builder.ensureAccountIsOnboarded([Role.HERMES]);
-  const worker = new HermesWorker(builder.dataModelEngine, config, logger);
+  const strategy = await loadStrategy(config.uploadStrategy);
+  const worker = new HermesWorker(builder.dataModelEngine, config, strategy, logger);
   await worker.start();
 }
 
+async function loadStrategy(uploadStrategy) {
+  const HermesUploadStrategy = require(`./workers/hermes_strategies/${uploadStrategy}`).default;
```

Comment:
Let's maybe keep whole path in config

Reply:
Putting all strategies in one directory requires some discipline when managing node. Reduces mess and simplifies config. I'd leave it like this.

Reply:
If someone starts it from docker as a black box it requires some black magick to put there